module/ldbc-connector/shared/src/main/scala/ldbc/connector/authenticator/AuthenticationPlugin.scala

```diff
@@ -6,8 +6,46 @@
 
 package ldbc.connector.authenticator
 
-trait AuthenticationPlugin:
+import java.nio.charset.StandardCharsets
+
+import scodec.bits.ByteVector
+
+import cats.syntax.flatMap.toFlatMapOps
+import cats.syntax.functor.toFunctorOps
+
+import cats.effect.Concurrent
+
+import fs2.hashing.HashAlgorithm
+import fs2.hashing.Hashing
+import fs2.Chunk
+trait AuthenticationPlugin[F[_]: Hashing](using F: Concurrent[F]):
```

Comment:
I feel this `AuthenticationPlugin` should not have `Hashing` and `Concurrent` constraints.
In order to do that, `hashPassword` implementation should be moved somewhere else.

Reply:
Until now, implementations differed by platform, so they were grouped together, but if they can be standardized by using fs2, it would be better to implement them separately, as you say.